buildSrc/src/main/kotlin/com.hedera.hashgraph.java-conventions.gradle.kts

```diff
@@ -0,0 +1,52 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+plugins {
+    java
+    jacoco
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+    maven {
+        url = uri("https://repo.maven.apache.org/maven2/")
+    }
+
+    maven {
+        url = uri("https://jcenter.bintray.com/")
+    }
+
+    maven {
+        url = uri("https://art.swirlds.com/artifactory/swirlds")
+    }
+}
+
+dependencies {
+    implementation("com.hedera.hashgraph:sdk:2.19.0")
+}
+
+group = "com.hedera.hashgraph"
+version = "0.12.2"
+java.sourceCompatibility = JavaVersion.VERSION_17
+
+//publishing {
+//    publications.create<MavenPublication>("maven") {
+//        from(components["java"])
+//    }
+//}
```

Comment:
Rather than committing commented out code, let's just leave it out

Reply:
Right, will do.